app/src/lib/git/status.ts

```diff
@@ -198,3 +158,97 @@ export async function getStatus(
     workingDirectory,
   }
 }
+
+/** reducer(ish) to create the map of file change statuses
```

Comment:
I think we can be a bit more expressive here, as `addEntryToFiles` doesn't reflect the fact that the original function did skip over some entries. Here's a quick something I put together to illustrate what I was hoping for:

```ts
/**
 *
 * Build collection of working directory changes using list of known
 * conflicted files.
 *
 * Map is used here to maintain insertion order.
 */
function parseStatusEntry(
  files: Map<string, WorkingDirectoryFileChange>,
  entry: IStatusEntry,
  filesWithConflictMarkers: Set<string>
) {
...
```

This moves the comment on line 130 closer to where it's being used.